cashews/serialize.py

```diff
@@ -40,21 +41,35 @@ async def _get_value(self, value, key, default=None):
             await super().delete(key)
             return default
 
+    def _get_value_without_signature(self, value: bytes, key: str) -> bytes:
+        if self._hash_key:
+            try:
+                sign, value = value.split(b"_", 1)
+            except ValueError:
+                return value
```

Comment:
in this case we can't just return a value, because then it will pass to by pickle without sign checking - so we will have a security issue. 
There are 2 ways to solve:
- remake that function to return unpickled data
- reraise the error here and catch it in  _process_value to return value without pickle